src/components/Image/Image.tsx

```diff
@@ -37,7 +37,13 @@ class Image extends UIComponent<any, any> {
   }
 
   renderComponent({ ElementType, classes, rest }) {
-    return <ElementType {...rest} className={classes.root} />
+    return (
+      <ElementType
+        {...this.accBehavior.generateAriaAttributes(this.props, this.state)}
+        {...rest}
+        className={classes.root}
+      />
+    )
```

Comment:
seems like there is no imports and behavior logic here, can you please add it? It might be me that missed it in previous commits